cocos/2d/CCFontFNT.cpp

```diff
@@ -641,10 +701,22 @@ FontAtlas * FontFNT::createFontAtlas()
         
         tempDefinition.offsetX  = fontDef.xOffset;
         tempDefinition.offsetY  = fontDef.yOffset;
-        
-        tempDefinition.U        = tempRect.origin.x + _imageOffset.x;
-        tempDefinition.V        = tempRect.origin.y + _imageOffset.y;
-        
+
+        const float left = _imageRectInPoints.origin.x;
+        const float right = (_imageRectInPoints.origin.x + rw);
+        const float top = _imageRectInPoints.origin.y;
```

Comment:
Define these variables outside of `for` is better.

Reply:
You're right, I missed that completely. I'll sort it out now.